nautobot/core/templates/graphene/graphiql.html

```diff
@@ -116,6 +116,46 @@
   </script>
   <script src="{% static 'graphene_django/graphiql.js' %}"></script>
   <script>
+  {% if perms.extras.view_graphqlquery %}
+    document.addEventListener('DOMContentLoaded', function() {
+      queries_tab = `
+        <button class="toolbar-button" id="navbar_queries">
+          <div>
```

Comment:
Normally I believe it's not correct to put a `div` inside a `button`. After a bit of experimenting, the following seems to me align more closely with the [Bootstrap docs](https://getbootstrap.com/docs/3.3/components/#dropdowns-example), still work correctly, and be a bit simpler besides:

```html
<div class="btn-group" role="button">
    <button class="toolbar-button dropdown-toggle" aria-haspopup="true" aria-expanded="false" data-toggle="dropdown">
        <span>Queries </span>
        <span class="caret"></span>
    </button>
    <ul class="dropdown-menu" id="graphql-queries-menu">
        <li class="dropdown-header">Saved Queries</li>
        {% for query in saved_graphiql_queries %}
            ....
        {% endfor %}
        <li class="divider" role="separator"></li>
        <li>
            <a onclick="populate_model()" data-toggle="modal" data-target="#GraphQLQueries_form" title="Query Form">Save Current Query As...</a>
        </li>
    </ul>
</div>